apps/full-site-editing/full-site-editing-plugin/common/data-stores/domain-suggestions.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * External dependencies
+ */
+import { DomainSuggestions } from '@automattic/data-stores';
+
+export const DOMAIN_SUGGESTIONS_KEY = DomainSuggestions.register( { vendor: 'variation2_front' } );
```

Comment:
@daledupreez We'll be shipping this to WordPress.com Simple and Atomic sites. We definitely won't be able the Calypso file that sets the vendor variation there.

Do we go with something static (like the PR does right now)? We could use something on Simple sites that could be updated quickly and easily, but Atomic sites will be another matter 😞

Reply:
For now, let's stick with the current hard-coded value. Most of our testing will be during NUX, so shouldn't be needed for post-signup use cases.

Reply:
@daledupreez Thanks! Do you see this is something we should deal with in future or are we good to do this and forget? 😁 

I'm thinking if this (and few other things I have in mind) would be good case to ask for more robust backend A/B testing tooling.

Reply:
We can leave this as-is for the time being. The real question here is that it's generally been easier to drive A/B tests from Calypso (without directly involving the back end), so that's how this has been done in the past.

If we're looking at a slower-changing client implementation in FSE, then it may make sense to change tack when we need to.